monai/data/dataset.py

```diff
@@ -515,6 +537,9 @@ class CacheDataset(Dataset):
     ``RandCropByPosNegLabeld`` and ``ToTensord``, as ``RandCropByPosNegLabeld`` is a randomized transform
     and the outcome not cached.
 
+    If want to update the input data and recompute cache content during training, just call `update_data()`
```

Comment:
Same with this sentence and following ones, it can be condensed down a bit.

Reply:
Sure, updated all related in the latest commit.

Thanks.